contrib/sssd.spec.in

```diff
@@ -64,8 +64,8 @@ Requires: sssd-common = %{version}-%{release}
 Requires: sssd-ipa = %{version}-%{release}
 Requires: sssd-krb5 = %{version}-%{release}
 Requires: sssd-ldap = %{version}-%{release}
-Recommends: sssd-proxy = %{version}-%{release}
-Recommends: logrotate
+Suggests: sssd-proxy = %{version}-%{release}
+Suggests: logrotate
```

Comment:
Hi,

maybe we can even drop `logrotate` but I'm fine with `Suggests` for the time being.

I would use `Requires` for `sssd-proxy` since there are valid use cases and it does not pull in extra dependencies like e.g. `sssd-ad`.

bye,
Sumit

Reply:
`logrotate` was added in 7fbc7e3ffb7a5c0090bb2091011762dabf1f512f IMO, makes sense.
IIRC, there was a BZ, but can't find it now...

Changed `sssd-proxy`.